frontend/app/routes/_gcweb-app.data-unavailable.tsx

```diff
@@ -29,8 +29,31 @@ export default function DataUnavailable() {
   const { SCCH_BASE_URI } = getClientEnv();
   return (
     <>
-      <p>{t('service-unavailable')}</p>
-      <InlineLink to={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })}>{t('gcweb:header.menu-dashboard.text')}</InlineLink>
+      <p className="mb-6">
+        {t('service-eligiblle')} (
+        <InlineLink id="do-you-qualify-link" to={t('do-you-qualify.href')}>
+          {t('do-you-qualify.text')}
+        </InlineLink>
+        )
+      </p>
+      <p className="mb-6">
+        {t('more-information')} (
+        <InlineLink id="how-to-apply-link" to={t('how-to-apply.href')}>
+          {t('how-to-apply.text')}
+        </InlineLink>
+        )
+      </p>
+      <p className="mb-6">
+        {t('other-enquiry')} (
+        <InlineLink id="contact-us-link" to={t('contact-us.href', { baseUri: SCCH_BASE_URI })}>
+          {t('contact-us.text')}
+        </InlineLink>
+        )
+      </p>
+      <p className="mb-8">{t('service-delay')}</p>
+      <InlineLink id="dashboard-link" to={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })}>
+        {t('gcweb:header.menu-dashboard.text')}
+      </InlineLink>
```

Comment:
Can we remove the references to the `gcweb` i18n namespace completely from this page and add a dashboard text/href i18n key in the `data-unavailable` namespace instead?

Reply:
fixed